Flutter/pdf-viewer/getting-started.md

```diff
@@ -160,6 +160,61 @@ Widget build(BuildContext context) {
 {% endhighlight %}
 {% endtabs %}
 
+### How to load document from base64 string?
+
+To load a document from a base64 string, you can convert the base64 string to a [`Uint8List`](https://api.flutter.dev/flutter/dart-typed_data/Uint8List-class.html) using the [base64Decode](https://api.flutter.dev/flutter/dart-convert/base64Decode.html) method and pass it to the [SfPdfViewer.memory](https://pub.dev/documentation/syncfusion_flutter_pdfviewer/latest/pdfviewer/SfPdfViewer/SfPdfViewer.memory.html) constructor. The following code example explains the same.
```

Comment:
Why this linked API alone is used inside `quote`.

Reply:
It was a mistake, and it is removed